packages/odyssey-react/.storybook/decorators/index.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import withGlobalReset from './withGlobalReset';
```

Comment:
minor, but consider the single line syntax for this:

```
export { default as withGlobalReset } from './withGlobalReset'
```

or even better just drop the default export entirely and remove the alias to simplify:

```
export { withGlobalReset } from './withGlobalReset'
```